config/environments/test.rb

```diff
@@ -1,6 +1,11 @@
 PrisonVisits2::Application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
+  config.action_mailer.smtp_settings = {
+    user_name: 'test_smtp_username',
+    password: 'test_smtp_password'
+  }
```

Comment:
When running tests, they already mock these settings, so we shouldn't need this to be set, I don't think?